src/main/java/duke/task/Deadline.java

```diff
@@ -0,0 +1,60 @@
+package duke.task;
+
+import java.time.LocalDateTime;
+
+/**
+ * Class for deadline tasks.
+ */
+public class Deadline extends Task {
+    /**
+     * Variable for date and time.
+     */
+    private LocalDateTime dateTime;
+
+    /**
+     * Constructor for event.
+     *
+     * @param description description of event
+     * @param date        date and time of event
+     */
+    public Deadline(String description, String date) {
+        super(description);
+        String[] dateSplit = date.split("\\s");
```

Comment:
I think the use of LocalDateTime.parse & DateTimeFormatter.ofPattern("dd/MM/yyyy HH:mm") could really help to simplify your code here!